runtime/j9vm/exports.cmake

```diff
@@ -403,6 +402,12 @@ if(NOT JAVA_SPEC_VERSION LESS 19)
 	)
 endif()
 
+if(JAVA_SPEC_VERSION LESS 20)
+	jvm_add_exports(jvm
+		_JVM_StopThread@12
+	)
+endif()
+
 if(NOT JAVA_SPEC_VERSION LESS 20)
```

Comment:
These three lines could be replaced by `else()`.

Reply:
Fixed.